MaxText/configs/base.yml

```diff
@@ -84,7 +84,7 @@ logits_dot_in_fp32: True  # whether to use fp32 in logits_dense or shared_embedd
 remat_policy: 'full'
 scan_layers: True
 param_scan_axis: 1
-attention: 'flash' # Supported attention: dot_product, flash, gpu_flash_xla, gpu_flash_triton
+attention: 'dot_product' # Supported attention: dot_product, flash, cudnn_flash_te
```

Comment:
Can we leave this as dot_product by default to avoid churn? For your tests you can override via CLI or env var.

(I know this is lame. I have a change that switches to "recommended" as the default that dispatches intelligently.)

Reply:
Hi @rwitten, it is set as `dot_product` in the PR currently. Did you mean to keep the default as `flash`?